### DISCUSSION TITLE
Problem for change Thruk URL

### DISCUSSION THREAD

**Discussion by samna88** [sni/thruk#1171] [Q&A]
Has anyone had the need to change the thruk url?

The problem is that the documentation for it is very scarce, it tells you to modify the url_prefix

![image](https://user-images.githubusercontent.com/102529906/161505966-203a9688-7f8f-4dc5-8be8-2c251e2786d7.png)

**Comment by sni** [comment]
in short, would advise you to not change the url. If you really need to, change the `url_prefix` setting and more important, change the apache configuration and rewrite rules.

**Reply by samna88** [reply]
thanks for your reply
